clio-client/src/main/scala/org/broadinstitute/clio/client/commands/ClioCommand.scala

```diff
@@ -50,6 +50,7 @@ sealed abstract class MoveCommand[TI <: TransferIndex](val index: TI)
     extends ClioCommand {
   def key: index.KeyType
   def destination: URI
+  def newSamplePrefix: Option[String] = None
```

Comment:
The name `new` is used here but it looks like this is just a prefix. Since it's not replacing an `old` prefix, it doesn't need `new` to disambiguate what it is. All sample prefixes in this commit are `new`.

Reply:
I like it. Brain was stuck on new-base-name from the ticket.